listener/listener.go

```diff
@@ -22,6 +24,16 @@ func configure() {
 	evalTopic := utils.GetenvOrFail("EVAL_TOPIC")
 
 	evalWriter = mqueue.WriterFromEnv(evalTopic)
+
+	parseValidReporters()
+}
+
+func parseValidReporters() {
```

Comment:
Can't we get the DB PR merged and use database to store list of valid reporters ? This duplicates the number of places, where we store list of valid reporters.

Reply:
Yes, it's done in the second PR.

Reply:
But that PR keeps the `parseValidReporters` method. It should then remove it, so that  there is only 1 source of this data.

Reply:
Yes, but it loads it from db: https://github.com/RedHatInsights/patchman-engine/pull/338/files so there IS only one source of reporters set.